Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -453,6 +458,56 @@ public HttpResponseMessage RemoveInsecureThirdCountry(int id, [FromBody] SingleV
                 .Match(_ => Ok(), FromOperationError);
         }
 
+        [HttpGet]
+        [Route("{id}/data-processing-registration-options")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage GetDataProcessingRegistrationOptions(int id, [FromUri] string nameQuery = null, [FromUri] int pageSize = 25)
+        {
+            return _dataProcessingRegistrationApplicationService
+                .GetDataProcessingRegistrationOptionsWhichCanBeAssigned(id)
+                .Select<DataProcessingOptionsDTO>(result => new DataProcessingOptionsDTO()
+                {
+                    roles = ToDTOs(result.DataProcessingRegistrationRoles, result.Registration.OrganizationId).ToList(),
+                    dataResponsibleOptions = ToDTOs(result.DataProcessingRegistrationDataResponsibleOptions).ToList()
+                })
+                .Match(Ok, FromOperationError);
+        }
+
+        [HttpPatch]
+        [Route("{id}/data-responsible")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage PatchDataResponsible(int id, [FromBody] SingleValueDTO<int?> dataResponsibleId)
+        {
+            if (dataResponsibleId == null)
+                return BadRequest("dataResponsibleId must be provided");
+
+            return _dataProcessingRegistrationApplicationService
+                .UpdateDataResponsible(id, dataResponsibleId.Value)
+                .Match(_ => Ok(), FromOperationError);
+        }
+
+        [HttpPatch]
+        [Route("{id}/data-responsible-remark")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage PatchDataResponsibleRemark(int id, [FromBody] SingleValueDTO<string> remark)
+        {
+            if (remark == null)
+                return BadRequest("dataResponsibleId must be provided");
```

Comment:
brug $"{nameOf(remark)} must be provided". Så fanger compileren selv dine copy-paste fejl til en anden gang